addons/surgery/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -112,4 +118,74 @@ class ACE_Medical_Treatment_Actions {
         callbackProgress = QFUNC(npwtTreatmentProgress);
         callbackSuccess = "";
     };
+    class UltraAssessment: BasicBandage {
+        displayName = CSTRING(Ultra_Use);
+        displayNameProgress = CSTRING(Ultra_Action);
+        category = "surgery";
+        treatmentLocations = QGVAR(surgicalLocation);
+        allowedSelections[] = {"Body"};
+        allowSelfTreatment = 0;
+        medicRequired = QGVAR(surgicalAction_MedLevel);
+        treatmentTime = QGVAR(intermediateTime);
+        items[] = {"kat_ultrasound"};
+        condition = QUOTE(!(_patient getVariable [ARR_2(QQGVAR(imaging),false)]));
+        consumeItem = 0;
+        callbackSuccess = QFUNC(ultraAssessment);
+    };
+    class ReboaPlacement: BasicBandage {
+        displayName = CSTRING(Reboa_Use);
+        displayNameProgress = CSTRING(Reboa_Action);
+        category = "surgery";
+        treatmentLocations = QGVAR(surgicalLocation);
+        allowedSelections[] = {"LeftLeg"};
+        allowSelfTreatment = 0;
+        medicRequired = QGVAR(surgicalAction_MedLevel);
+        treatmentTime = QGVAR(intermediateTime);
+        items[] = {"kat_reboa"};
+        condition = QUOTE((_patient getVariable [ARR_2(QQGVAR(imaging),false)]) && (!(_patient getVariable [ARR_2(QQGVAR(reboa),false)])));
+        consumeItem = 1;
+        callbackSuccess = QFUNC(reboaApply);
+    };
+    class ReboaAdvancement: BasicBandage {
+        displayName = CSTRING(Reboa_Deep_Use);
+        displayNameProgress = CSTRING(Reboa_Deep_Action);
+        category = "surgery";
+        treatmentLocations = QGVAR(surgicalLocation);
+        allowedSelections[] = {"LeftLeg"};
+        allowSelfTreatment = 0;
+        medicRequired = QGVAR(surgicalAction_MedLevel);
+        treatmentTime = QGVAR(intermediateTime);
+        items[] = {"kat_ultrasound"};
+        condition = QUOTE(_patient getVariable [ARR_2(QQGVAR(reboa),false)]);
+        consumeItem = 0;
+        callbackSuccess = QFUNC(reboaDeepApply);
+    };
+    class ReboaRemoval: BasicBandage {
```

Comment:
i would consider just inheriting from ReboaPlacement to reduce junking file with repetetive stuff